css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,53 @@
+@import "../page-vars";
+
+.ampstart-headerbar {
+  justify-content: space-between
+
+  height: var(--layout-header-height-mobile);
+
+  /* Override ampstart default box shadow styles */
+  box-shadow: none;
+
+  /* header is fixed so should be moved onto a new GPU layer */
+  will-change: transform;
+}
+
+.ampstart-headerbar-home-link {
+  position: absolute;
```

Comment:
@camelburrito OK we've taken onboard your comments. In our latest commits we have removed the `absolute` positioning and positioned the elements in the navbar using bass css columns. It does mean adding a few more classes via the JSON (See changes to the `ampstart-navbar` and `data.json`, but this keeps the navbar customizable and keeps the title centered.